### ISSUE TITLE
Registration issue after top-up (dVPN)

### ISSUE THREAD

**Issue by adinetech** [mysteriumnetwork/node#5298]
**Describe the bug**
Registration does not occur after a top-up and the MYST is in the Internal channel identity.

**Expected behavior**
It should register the identity once the top-up is completed, at least it should after restarting the app but nothing works.

**Desktop:**
 - OS: Windows 10
 - App Version: 10.11.2

**Comment by adinetech** [comment]
https://github.com/mysteriumnetwork/mysterium-vpn-desktop/issues/366

**Comment by etherunit** [comment]
I topped off the account twice. For example, the second top off was

CoinGate Bitcoin Payment Processing
CoinGate Order ID #12669559
BlockDev AG
3.99 USD
0.0001719 BTC

2022-07-21 01:45:09 UTC

2022-07-21 02:19:15 UTC



Using the VPN on Mac OS 10.13.6

Latest version of the VPN software 10.12.4

It is stuck on the screen that says "Top up your wallet now to complete the registration". No other menus or buttons are available.

Also for reference, below are some more details of the second top off. I used the bitcoin option.

cfe4144ce549cf7dd0da57a417e4e405889cba295152590c6c66b2fc3a25bc4c
Destination address: 3Kx8D6uQnE8WR8ZGktCjdoue5hcDf3b7rY
From address: 325NHodL3u5t5CTFa78aY3Xyj6asat1BsX
0.00017190 BTC
